### PR TITLE
Questions

### PR DESCRIPTION
**Pull request by ragini07**
get quiz by category
get quiz questions and play quiz

### CODE REVIEW

File: src/Components/Header/Header.jsx [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+      {/* <ul className="right-menu"> 
             <li>
                 <i className="fa fa-user fa-2x"></i>
            </li>
-        </ul>
+        </ul> */}
     </nav>
-  )
+  );
 }
```
Comment (JowelTisso) [inline src/Components/Header/Header.jsx:24]: Remember to remove unused code

**Review by JowelTisso — APPROVED** [review]
* Clean and modular code
* Good small PR
* Rest looks good to merge

**Review by Ash-D23 — APPROVED** [review]
Code Looks clean
UI looks good
Good naming Convention

**Review by rahulmistry751 — APPROVED** [review]
Good code structuring

**Pull request merged**